docs/draft/6-payloads.md

```diff
@@ -191,6 +195,22 @@ message ImageMessage {
 }
 ```
 
+##### Audio content type
+
+A `ChatMessage` with `AUDIO` `Content/Type` MUST also specify the `payload` of the audio
+and the `type`
+
+```protobuf
+message AudioMessage {
+  bytes payload = 1;
```

Comment:
Any max size?

Reply:
No, same as images, we don't limit size at this layer, the max size that is currently valid for any waku message (1MB) still stands.
If we want to allow users to send more data, it will not be handled at this layer but closer to the transport (by splitting the whole protobuf into chunks)

Reply:
right, forgot we dealt with that above already